protocols/kad/src/behaviour.rs

```diff
@@ -185,12 +185,12 @@ impl Default for KademliaConfig {
             kbucket_pending_timeout: Duration::from_secs(60),
             query_config: QueryConfig::default(),
             protocol_config: Default::default(),
-            record_ttl: Some(Duration::from_secs(36 * 60 * 60)),
+            record_ttl: Some(Duration::from_secs(48 * 60 * 60)),
```

Comment:
Note @melekes and @kpp that this invalidates the assumptions in Substrate's Authority Discovery.

https://github.com/paritytech/substrate/blob/8d11cec9713a93b9f6bee1b29fde840da5ba56e0/client/authority-discovery/src/lib.rs#L88-L109

Either you go with the new defaults and update Authority Discovery, or you override the defaults of libp2p.

Reply:
Right. Thanks for mentioning 👍